package.json

```diff
@@ -10,13 +10,19 @@
         "@testing-library/jest-dom": "^5.11.4",
         "@testing-library/react": "^11.1.0",
         "@testing-library/user-event": "^12.1.10",
+        "@types/draftjs-to-html": "^0.8.0",
         "@types/jest": "^26.0.15",
         "@types/node": "^12.0.0",
         "@types/react": "^17.0.0",
         "@types/react-dom": "^17.0.0",
+        "@types/react-draft-wysiwyg": "^1.13.2",
+        "draft-js": "^0.11.7",
+        "draftjs-to-html": "^0.9.1",
+        "draftjs-to-markdown": "^0.6.0",
         "jwt-decode": "^3.1.2",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
+        "react-draft-wysiwyg": "^1.14.7",
```

Comment:
We need all of these dependencies to make this work?

Reply:
I had removed unused dependencies.

Reply:
![image](https://user-images.githubusercontent.com/58205909/121802408-503f7a80-cc66-11eb-810b-35f018309811.png)